server/src/common/actions/organismes/organismes.formations.actions.ts

```diff
@@ -29,43 +28,43 @@ export const getFormationsTreeForOrganisme = async (uai) => {
   let nbFormationsCreatedForOrganisme = 0;
   let nbFormationsNotCreatedForOrganisme = 0;
 
-  if (catalogFormationsForOrganisme?.length) {
-    await asyncForEach(catalogFormationsForOrganisme, async (currentFormation) => {
-      let currentFormationId;
-
-      // Récupération de la formation du catalogue dans le TDB, si pas présente on la crée
-      // On compte les formations créées / non créées (erreur)
-      const formationFoundInTdb = await getFormationWithCfd(currentFormation.cfd);
-      if (!formationFoundInTdb) {
-        try {
-          currentFormationId = await createFormation(currentFormation);
-          nbFormationsCreatedForOrganisme++;
-        } catch (error) {
-          nbFormationsNotCreatedForOrganisme++;
-        }
-      } else {
-        currentFormationId = formationFoundInTdb._id;
+  for (const currentFormation of catalogFormationsForOrganisme) {
```

Comment:
remplacement du asyncForEach par for of

Reply:
Oui car ce helper ne sert à rien et fait exactement la même chose qu'un for of qui contient un await. :upside_down_face:

Reply:
👉🏼 https://github.com/mission-apprentissage/flux-retour-cfas/issues/2973 :)